_pages/en_US/installing-unlaunch.md

```diff
@@ -84,7 +86,7 @@ This section is optional and only serves for keeping your SD card tidy of files
 {: .notice--primary}
 
 - Delete the `sd:/private/ds/app/484E494A/pit.bin` file from your SD card
-   - If you used another exploit, delete the relevant files concerning that exploit instead
+   - If you used another exploit, delete the files from that exploit instead
```

Comment:
Is it really needed to say this? how would other exploits interfere?

Reply:
This is for cleaning up your SD after you install Unlaunch, they don't interfere with anything, but good to let people know that they have different files to clean up if they didn't use Memory Pit